C7/Game.cs

```diff
@@ -618,4 +623,12 @@ private void OnBuildCity(string name)
 	{
 		new MsgBuildCity(CurrentlySelectedUnit.guid, name).send();
 	}
+
+	private void OnSaveGame()
+	{
+		GD.Print("Saving Game...");
```

Comment:
It would be convenient to print out the save file location here too, since we don't yet give the player the option to choose it. I ran this branch to test it and it was not obvious to me that the save file would overwrite c7-static-map-save.json.